packages/dx-chart-core/src/plugins/axis/computeds.ts

```diff
@@ -41,14 +46,40 @@ const createVerticalOptions = (position, tickSize, indentFromAxis) => {
   };
 };
 
-export const axisCoordinates = ({
+type AxisCoordinatesArg = {
+  readonly scaleName: string;
+  readonly scale: Scale;
+  readonly position: string;
+  readonly tickSize: number;
+  readonly tickFormat: TickFormatFn;
+  readonly indentFromAxis: number;
+};
+type Tick = {
+  readonly key: string;
+  readonly x1: number;
+  readonly x2: number;
+  readonly y1: number;
+  readonly y2: number;
+  readonly xText: number;
+  readonly yText: number;
+  readonly dominantBaseline: string;
+  readonly textAnchor: string;
+  readonly text: string;
+};
+type AxisCoordinatesResult = {
+  readonly ticks: ReadonlyArray<Tick>;
+  readonly sides: Readonly<[number, number]>;
```

Comment:
These readonly modifiers can be removed since PureComputed makes argument types deep readonly